### DISCUSSION TITLE
Very tired and very pissed at the moment - FCPX system crashes when I try to drag prepared braw file to library

### DISCUSSION THREAD

**Discussion by ChristinaCookLee** [latenitefilms/BRAWToolbox#125] [Support] [answered]
Bought Toolbox today. Was all excited for an easy solution after spending hours trying to convert a stinking file manually. I'm sick of reading and watching tutorials about Toolbox and working for hours trying different things that all end up with the same result...crash. I have a massive crash report trail of my failed efforts today that I would like to send so you can tell me what is going on. Please tell me where to send. Must use the client's BMPC4K or I wouldn't even bother at this point. Need to get this done. christinacooklee@gmail.com. Please email me. I hate forums and have invested enough time on this without having to check back here to see if a solution has been given. Thank you.

**Comment by latenitefilms** [comment, marked as answer]
Sorry to hear you're having issues! I've just emailed you.

FWIW - you can find our support email here: 

https://brawtoolbox.io/support/

For anyone else who comes across this...

Are you running Final Cut Pro 10.6.5 as per the system requirements?

https://brawtoolbox.io/system-requirements/

Did you follow the Installation Instructions?

https://brawtoolbox.io/installation/

If you need to send us crash logs, you can find all the details here:

https://brawtoolbox.io/troubleshooting/#ive-run-into-a-bug-where-can-i-find-the-log-files

**Comment by ChristinaCookLee** [comment]
Hi Chris,

I wanted to let you know that I think I figured out the problem. It was a
space issue. The file I was working with was large--too large for the
amount of space available on the hard drive--and I didn't realize it in my
exhausted state. Final Cut was simply crashing when I tried to pull
the processed file in. Would have been nice if some code had been written
in to allow for a warning, but after all, it seemed to have been tossed to
my desktop?? Kinda crazy. Anyway. I apologize that I was tired and pissed
last night. Things are looking better today and I have hope. I appreciate
the Toolbox and will be using it regularly. Before I figured out what was
going on today, I wondered why your plugin doesn't simply convert the file
to a selection of formats and allow the user to save it to anywhere they
like and then just pull it into FCP or whatever. It's nice having the
extension controls within a program, but the file itself seems like it
could be saved elsewhere and still used in your extension. Just a thought.

All the BEST,

* <(941)%20224-5621>*

<http://christinacooklee.com/>

*Website:* *ChristinaCookLee.com* <http://christinacooklee.com/>

*Email:* ***@***.**** ***@***.***>
*Phone:*
*(941) 224-5621 <(941)%20224-5621>*

**Reply by latenitefilms** [reply]
BRAW Toolbox doesn't move or copy any files. When you import a BRAW file via the Workflow Extension, all we're doing is "linking" to that file, where ever it lives on your machine. We don't move it or copy it. If your BRAW file is on external hard drive, when you import to BRAW Toolbox, we simply "link" to that file. The "Copy to library storage location" and "Leave files in place" settings in the Final Cut Pro Import window have no impact on BRAW Toolbox.

To clarify - we ALWAYS read the BRAW file natively and directly - no transcoding is happening. If you want to work with ProRes, you're better off using EditReady, DaVinci Resolve or Color Finale Transcoder to make ProRes copies of the BRAW.

Essentially, the BRAW Toolbox Workflow Extension is basically just a FCPXML generator and audio ripper. All it does it allow you to preset the RAW values, and then when you click **Prepare BRAW Files**, it exports the audio from the BRAW files, and generates a FCPXML that you drag into Final Cut Pro.

Your crash log was a Final Cut Pro crash - nothing to do specifically with BRAW Toolbox (aside from the fact that BRAW Toolbox created the FCPXML). If Final Cut Pro did indeed crash because of lack of hard drive space on your system drive, then this is a Final Cut Pro bug - nothing to do with BRAW Toolbox.

Whilst I've got you... a few things to keep in mind when using BRAW Toolbox:

- Please turn on **Create Multicam Clips** (instructions [here](https://brawtoolbox.io/faq/#i-dont-like-synchronised-clips-can-i-import-a-multicam-clips-instead)). Final Cut Pro currently has some issues with Synchronised Clips.
- You can get a small performance gain if you **Grant Sandbox Access** to the drive that contains your BRAW footage (instructions [here](https://brawtoolbox.io/collaboration/)).
- If you need to send a FCPXML to DaVinci Resolve or elsewhere, you have to use [this toolbox](https://brawtoolbox.io/toolbox/#convert-project-to-resolve-friendly-fcpxml).

There's a few videos and lots of documentation on the website that discusses everything:

https://brawtoolbox.io

Happy cutting! Feel free to reach out if you have any further questions!
